tests/x11/reboot_gnome.pm

```diff
@@ -40,12 +40,19 @@ sub run() {
 
         send_key "ret";
 
-        wait_still_screen 8, 8;    # wait 8 seconds to make authentication window disappear after successful authentication
```

Comment:
I think it should not happen, s390x/zkvm doesn't have floating performance as qemu
But I will feel safer with 10 seconds sleep

http://10.100.12.155/tests/3688
http://10.100.12.155/tests/3689